### ISSUE TITLE
On doctrine:schema:update every time alter table

### ISSUE THREAD

**Issue by anydasa** [ramsey/uuid-doctrine#64]
Please add to src/UuidType.php the code to avoid this problem 

```
/**
 * @param AbstractPlatform $platform
 *
 * @return array
 */
public function getMappedDatabaseTypes(AbstractPlatform $platform)
{
    return [self::NAME];
}
```

**Comment by skillbox-koutja** [comment]
I have same problem with postgres.

**Comment by ramsey** [comment]
I'm sorry it's taken so long for me to respond to this. Can you provide more information so I can reproduce this? Thanks!

I'm trying to find examples in doctrine/dbal that use `getMappedDatabaseTypes()`, and I can't find anything. Their `GuidType` and `StringType` do not implement this method, for example.

**Comment by gansky-alexander** [comment]
Hello, all ! 

Any updates about this issue ? Maybe someone can provide any advice?

**Comment by ramsey** [comment]
@gansky-alexander, I need more information about the issue. Are you able to provide more information?

**Comment by akondas** [comment]
Hi, I have similar issue. My current packages (symfony 5.3):
```
"doctrine/doctrine-bundle": "^2.1",
"doctrine/doctrine-migrations-bundle": "^3.0",
"doctrine/orm": "^2.7",
"ramsey/uuid-doctrine": "^1.6"
```

When I run `bin/console d:m:diff` each time alter table is added, but I always fix this with:
```php
$this->addSql('COMMENT ON COLUMN my_table.id IS \'(DC2Type:uuid)\'');
```

It looks as if this comment has stopped adding itself since some update :thinking:

**Comment by francescolaffi** [comment]
I reproduced the issue after updating to 1.7 and postgres, I think is caused by the removal of `\Ramsey\Uuid\Doctrine\UuidType::requiresSQLCommentHint` that is always needed for custom types, even if the platform support the type natively, otherwise doctrine cannot differentiate the custom type mapping to the native uuid type

**Comment by ramsey** [comment]
I've just released version 1.8.0, which includes #65 and #165. Thanks!
